index.js

```diff
@@ -1,6 +1,61 @@
 /* jshint node: true */
 'use strict';
 
+var path = require('path');
+var Funnel = require('broccoli-funnel');
+var mergeTrees = require('broccoli-merge-trees');
+var VersionChecker = require('ember-cli-version-checker');
+
 module.exports = {
-  name: 'ember-a11y'
+  name: 'ember-a11y',
+
+  init: function() {
+    if (this._super.init) {
+      this._super.init.apply(this, arguments);
+    }
+
+    var versionChecker = new VersionChecker(this);
+    versionChecker.for('ember-cli', 'npm').assertAbove('0.2.0');
+
+    var emberVersion = versionChecker.for('ember', 'bower');
+
+    if ((emberVersion.gt('2.9.0-beta') && emberVersion.lt('2.9.0')) || emberVersion.gt('2.10.0-alpha')) {
+      this.versionSpecificPath = path.join(this.root, 'version-specific-2.9');
```

Comment:
Let's name the paths based upon the feature (Glimmer, HTMLBars), not the version.